TOC.md

```diff
@@ -98,7 +98,7 @@
     - [Hybrid Topology](/hybrid-deployment-topology.md)
   - Install and Start
     - [Use TiUP](/production-deployment-using-tiup.md)
-    - [Deploy in Kubernetes](/tidb-in-kubernetes.md)
+    - [Deploy on Kubernetes](/tidb-in-kubernetes.md)
```

Comment:
https://github.com/ran-huang/docs/blob/on-k8s/br/backup-and-restore-faq.md?plain=1#L183

```
## How can I use BR for incremental backup on Kubernetes?
```
IMO, `environment` is redundant.

Reply:
done [8f5900f](https://github.com/pingcap/docs/pull/11224/commits/8f5900fda36a0f3b5b020800d0e8c2ebc3cb3d29)